docs/references/pysdk_api_reference.md

```diff
@@ -1706,38 +1706,49 @@ for question in questions:
 ## match tensor
 
 ```python
-table_object.match_tensor(vector_column_name, tensor_data, tensor_data_type, method_type, topn, extra_option)
+table_object.match_tensor(vector_column_name, tensor_data, tensor_data_type, method_type, extra_option)
 ```
 
-Builds a KNN tensor search expression. Find the top n closest rows to the given tensor according to chosen method.
-
-For example, find k most match tensors generated by ColBERT.
+Performs a k-nearest neighbor (KNN) tensor search on a specified column to identify the top n closest rows to the given tensor. Suitable for working with tensors generated by ColBERT.
 
 ### Parameters
 
 #### vector_column_name: `str`, *Required*
 
+A non-empty string indicating the name of the tensor column to search on.
 
 #### tensor_data: `list/np.ndarray`, *Required*
 
+The query vector data to compare against. This should be provided as a list or a two-dimensional NumPy array of numerical values.
 
 #### tensor_data_type: `str`, *Required*
 
+Specifies the data type of the tensor. Commonly used types (values) include:
+
+- `"float"`
+- `"uint8"`.
+
+#### method_type: `str`, *Required*
+
+A non-empty string indicating the ranking method. Currently, only `"maxsim"` is supported.
 
-#### method_type: `str`
+#### extra_option: `str`, *Required*
 
--  `'maxsim'`
+A non-empty string specifying the following search options:
 
-#### extra_option: `str` 
+- **"topn"**: `str`, *Required*  
+  Specifies the number of the most relevant rows to retrieve, e.g., `"topn=10"` to obtain the ten most relevant rows.
+- **EMVB index options**: `str`, *Optional*  
+  Used when an EMVB index is successfully created on the column being queried. Supported options:
+  - `"emvb_centroid_nprobe"`
```

Comment:
Each meaning of the parameter.

Reply:
nprobe: similar to nprobe in IVF
following parameters: chosen in experiment, may vary across different datasets